src/components/Profile/Settings/NotificationsComponent/NotificationHeader.js

```diff
@@ -1,41 +1,58 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import { toggleNotificationType } from '../../../../actions/notificationActions';
+import {
+  toggleNotificationType,
+  getNotificationConfiguration
+} from '../../../../actions/notificationActions';
 
-const NotificationHeader = ({ notification, toggleNotificationType }) => (
-  <div>
-    <h3>Notifications</h3>
-    <p>Select from where you would like receive your notifications</p>
-    {Object.keys(notification).map((key, index) => (
-      <div key={index}>
-        <input
-          type="checkbox"
-          key={index}
-          className="medium-margin"
-          data-test={key}
-          checked={notification[key].articles.show}
-          onChange={() => toggleNotificationType(key)}
-        />
-        {notification[key].alias}
-        {Object.keys(notification).length === index + 1 ? (
-          <div className="divider b-bottom-light-grey" />
-        ) : null}
+class NotificationHeader extends Component {
+  componentDidMount() {
+    const { getNotificationConfiguration } = this.props;
+    getNotificationConfiguration();
+  }
+
+  render() {
+    const { notification, toggleNotificationType } = this.props;
+    return (
+      <div>
+        <h3>Notifications</h3>
+        <p>Select from where you would like receive your notifications</p>
+        {Object.keys(notification).map((key, index) => (
+          <div key={index}>
+            <input
+              type="checkbox"
+              key={index}
+              className="medium-margin"
+              data-test={key}
+              checked={notification[key].articles.show}
+              onChange={() => toggleNotificationType(key)}
+            />
+            {notification[key].alias}
+            {Object.keys(notification).length === index + 1 ? (
+              <div className="divider b-bottom-light-grey" />
+            ) : null}
+          </div>
+        ))}
       </div>
-    ))}
-  </div>
-);
+    );
+  }
+}
 
 NotificationHeader.propTypes = {
   notification: PropTypes.object.isRequired,
```

Comment:
@MBenedicte, same thing here
```suggestion
  configuration: notificationReducer.config,
```